src/renderers/shared/stack/reconciler/ReactCompositeComponent.js

```diff
@@ -197,8 +196,11 @@ var ReactCompositeComponentMixin = {
 
     var Component = this._currentElement.type;
 
+    // Select the update queue associated to the transaction
```

Comment:
This comment repeats exactly what the line does—let’s remove it.
Or maybe change to something like “When rendering on the server, we want updates to be no-ops after mounting”.